src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -279,6 +279,10 @@ namespace OpenMS
       */
       extern OPENMS_DLLAPI const std::string   FRAGMENT_ERROR_MEDIAN_PPM_USERPARAM;
 
+      extern OPENMS_DLLAPI const std::string   FRAGMENT_ERROR_PPM_METAVALUE;
```

Comment:
don't want to be pickt but we use _USERPARAM
Is the FRAGMENT_ERROR_PPM_METAVALUE for a single peak or e.g. the mean? if it is the mean I would
put _MEAN_ similar as we did for   FRAGMENT_ERROR_MEDIAN_PPM_USERPARAM;

Reply:
Alright, I'll add _USERPARAM.

It isn't the mean over a whole spectrum though. The Metavalue stores a list of all ppms (one for each peak in the spectrum).